dashboard/package.json

```diff
@@ -41,7 +41,8 @@
     "react-syntax-highlighter": "^15.5.0",
     "recharts": "^2.3.2",
     "styled-components": "5.3.0",
-    "ts-proto": "^1.156.2"
+    "ts-proto": "^1.169.1",
+    "typescript": "5.4.2"
```

Comment:
Otherwise we'll encounter the following error.

```
     ▲ Next.js 14.1.0
     Skipping linting
     Checking validity of types ...
  --- stderr
  It looks like you're trying to use TypeScript but do not have the required package(s) installed.
  Please install typescript by running:
  	npm install --save-dev typescript
```

Not sure why it doesn't happen on docker builds.